app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts # user can have many posts
+  has_many :comments, through: :post # User can have 0 or more comments in one
```

Comment:
- [ ] you should set the foreign_key here as well

Reply:
Shouldn't the foreign key be found only in the corresponding migration file